modules/dhcp/providers/server/isc.rb

```diff
@@ -25,7 +32,6 @@ def delRecord subnet, record
       omcmd "open"
       omcmd "remove"
       omcmd("disconnect", msg)
-      subnet.delete record
     end
```

Comment:
It used to be, but it was an empty method, didn't think there was any point to keep it around.